### PR TITLE
docs: concepts (suggestions on #2390) 

### PR DESCRIPTION
**Pull request by desimone**
See commit messages for breakdown of changes. Should be git-cherry-pickable.

### CODE REVIEW

**Review by alexfornuto — COMMENTED** [review]
Apparently optipng isn't the best tool for the job anymore. I'll update my methods.

File: docs/enterprise/concepts.md [hunk 1]
```diff
@@ -1,18 +1,81 @@
...
+#### Admin
+
+An Admin user has permissions across all Namespaces. They can manage global settings, sessions, and service accounts, as well as view events and runtime data.
 
 ## Service Accounts
 
-Service accounts handle machine-to-machine communication from Pomerium to your Identity Provider (**IdP**) in order to retrieve and establish group membership. Configuration is largely dependent on the IdP, but is usually an API acccess token with sufficient privlidges to read users and groups.
+Service accounts handle machine-to-machine communication from Pomerium to your Identity Provider (**IdP**) in order to retrieve and establish group membership. Configuration is largely dependent on the IdP, but is usually an API access token with sufficient privileges to read users and groups.
+
+<!-- @travisgroth -- could you add some context in here? I think your PRD on service accounts would be super helpful and you know this concept best-->
```
Comment (alexfornuto) [inline docs/enterprise/concepts.md:78]: CC @travisgroth

File: docs/enterprise/concepts.md [hunk 2]
```diff
@@ -1,18 +1,81 @@
...
+
+#### Viewer
+
+A user with the Viewer role can:
+
+- view all resources in a Namespace (Routes, Policies, Certificates),
+- view traffic dashboard for routes in the Namespace, including child Namespaces
+- view the activity log for a namespace
+- view all namespaces (by list, not their contents) <!-- @bobby please double-check me here -->
+- query all directory data <!-- Copied from PRD, but unsure if implemented or what it does -->
```
Comment (alexfornuto) [inline docs/enterprise/concepts.md:64]: Please confirm/edit/qualify/remove

File: docs/enterprise/concepts.md [hunk 3]
```diff
@@ -1,18 +1,81 @@
...
+Consider this scenario: your organization has a security team managing high-level, course grain authorization controls. For example, they know that everyone with access to internal resources:
+
+   - has a `yourcompany.com` email account,
+   - isn't coming from a known bad actor IP address,
+
+Identity and access is organized and controlled with Identity and Access Management (**IAM**) teams in your Identity Provider (**IdP**). These users and groups are read by Pomerium from your IdP, and used to determine top-level Namespace organization.
+
+From there, Application owners are assigned to child Namespaces, and are given the flexibility to self-manage their own application routes and policies.
+
+<!-- @Bobby This is my attempt to  prose-ify your block.-->
```
Comment (alexfornuto) [inline docs/enterprise/concepts.md:49]: CC @desimone for review

File: docs/enterprise/concepts.md (hunk 2 above)
Comment (desimone) [reply docs/enterprise/concepts.md:64]: \cc @travisgroth

File: docs/enterprise/concepts.md [hunk 4]
```diff
@@ -1,18 +1,81 @@
...
+### Hierarchical Policy Enforcement
+
+Hierarchical policy lets administrators enforce high level authorization policy. Policies can be optional (self-select), or mandatory.
+
+Consider this scenario: your organization has a security team managing high-level, course grain authorization controls. For example, they know that everyone with access to internal resources:
+
+   - has a `yourcompany.com` email account,
+   - isn't coming from a known bad actor IP address,
+
+Identity and access is organized and controlled with Identity and Access Management (**IAM**) teams in your Identity Provider (**IdP**). These users and groups are read by Pomerium from your IdP, and used to determine top-level Namespace organization.
```
Comment (desimone) [inline docs/enterprise/concepts.md:45]: E.g. CFO has access to everything in the accounting namespace.

**Pull request merged**